sdcm/cluster.py

```diff
@@ -4894,17 +4894,38 @@ def reconfigure_scylla_monitoring(self):
                         f"[ -f /etc/rsyslog.d/scylla.conf ] || scylla_rsyslog_setup --remote-server {node.ip_address}"
                     ), verbose=True)
 
+    def _create_manager_prometheus_yaml(self, node):
+        manager_prometheus_port = self.params.get("manager_prometheus_port")
+        # Manager server prometheus:
+        manager_prometheus_target_list = [f"127.0.0.1:{manager_prometheus_port}"]
+        # Agents prometheus:
+        manager_prometheus_target_list.extend(
+            [f"{db_node.ip_address}:{manager_prometheus_port}" for db_node in self.targets["db_cluster"].nodes])
```

Comment:
are agents and server using the same port ?

Reply:
Yes - https://github.com/scylladb/scylla-cluster-tests/blob/master/sdcm/cluster.py#L1840